docs/config.md

```diff
@@ -48,6 +48,7 @@ All configuration files should be placed inside the application's configuration
 | `liked_icon`                         | the icon to indicate the liked state of a song                                           | `♥`                                                       |
 | `border_type`                        | the type of the application's borders                                                    | `Plain`                                                    |
 | `progress_bar_type`                  | the type of the playback progress bar                                                    | `Rectangle`                                                |
+| `album_format`                  | how the name of the album is formatted in the artist's album list                                                    | `YearName`                                                |
```

Comment:
as I mentioned before, don't need a config option for this. We should always include the year information.

Reply:
Sure! I removed the option.